### PR TITLE
picture, quote generator, background

### PR DESCRIPTION
**Pull request by apoorvadhanala**
Add picture, create random quote generator function, update background/introduction paragraph

image
image

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -8,11 +8,18 @@
...
+</p>
+<p>
+I was born and raised in Atlanta, Georgia and have a little sister. I have learned piano and the classical Indian dance form, Kuchipudi, since I was 5 years old; the skills I have learnt from these art forms, from patience and perseverance to responsibility and delicateness, are a big part of who I am today. Some of my favorite things to do are travel, take pictures, cook, and watch TV. I love traveling to new places and getting to see different parts of the world and I also love photography and taking pictures whether they are travel pictures or senior pictures. As for cooking and baking, some of my favorite things to experiment with are different types of pasta & sauces and also chocolate chip cookies! Whenever I watch TV, some of my go-to’s are Suits, Modern Family or Scandal. 
+ 
+Click on any of the links below to find out more about who I am whether it has to do with my professional experiences or more about my interests and hobbies! 
+</p>
+      <img src="/images/me.jpg" alt="Here's a picture of me!" width="200" height="300">
```
Comment (yjennywang) [inline portfolio/src/main/webapp/index.html]: I think setting width="200" and height="300" here is fine, but you could also consider making a class for it to keep the styling in CSS.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -8,11 +8,18 @@
...
-      <div id="greeting-container"></div>
+      <h1>Apoorva Dhanala's Portfolio</h1>
+      <p>Hi! My name is Apoorva Dhanala and I am a rising junior at the University of Georgia studying Computer Science with an emphasis in Artificial Intelligence. At UGA, I am founder and president of UGA’s chapter of Girls Who Code where we work with local, underprivileged Athens-Clarke County schools where UGA students volunteer weekly hoping to expose students to coding while closing the gender gap in technology. I am also involved with the Georgia Recruitment Team where I represent the admissions office as an ambassador at panels and tours, Athens Prison Tutorial where I tutor prison inmates at the Athens-Clarke County Jail who lack their high school diplomas and help them improve their skills in order to pass their GED examination, and with the Honors Program Student Council where I work to help bring academic and social opportunities to students in the Honors Program. Outside of classes and club involvement, I am a CURO Honors Research scholar where I have been involved with Linguistics and Natural Language Processing research and most recently, Android App development work.
```
Comment (yjennywang) [inline portfolio/src/main/webapp/index.html]: Could you break up the paragraphs into lines (no specific column limit, not 80ish is nice) to help with readability, please?

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -8,11 +8,18 @@
...
+</p>
+<p>
+I was born and raised in Atlanta, Georgia and have a little sister. I have learned piano and the classical Indian dance form, Kuchipudi, since I was 5 years old; the skills I have learnt from these art forms, from patience and perseverance to responsibility and delicateness, are a big part of who I am today. Some of my favorite things to do are travel, take pictures, cook, and watch TV. I love traveling to new places and getting to see different parts of the world and I also love photography and taking pictures whether they are travel pictures or senior pictures. As for cooking and baking, some of my favorite things to experiment with are different types of pasta & sauces and also chocolate chip cookies! Whenever I watch TV, some of my go-to’s are Suits, Modern Family or Scandal. 
+ 
+Click on any of the links below to find out more about who I am whether it has to do with my professional experiences or more about my interests and hobbies!
```
Comment (yjennywang) [inline portfolio/src/main/webapp/index.html]: Is this the start of a new paragraph? Do you need a  new <p> here?

**Review by yjennywang — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/photography.html [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+        <a href="images/stanley-4.jpg"><img src="images/stanley-4.jpg"/></a>
+        <a href="images/stanley-5.jpg"><img src="images/stanley-5.jpg"/></a>
+        <a href="images/stanley-6.jpg"><img src="images/stanley-6.jpg"/></a>
+        <a href="images/stanley-7.jpg"><img src="images/stanley-7.jpg"/></a>
+        <a href="images/stanley-8.jpg"><img src="images/stanley-8.jpg"/></a>
+        <a href="images/stanley-9.jpg"><img src="images/stanley-9.jpg"/></a>
+        <a href="images/stanley-10.jpg"><img src="images/stanley-10.jpg"/></a>
+        <a href="images/stanley-11.jpg"><img src="images/stanley-11.jpg"/></a>
+        <a href="images/stanley-12.jpg"><img src="images/stanley-12.jpg"/></a>
+        <a href="images/stanley-13.jpg"><img src="images/stanley-13.jpg"/></a>
```
Comment (sidharthp) [inline portfolio/src/main/webapp/photography.html:26]: These images aren't present in this pull request, which is causing a broken page - http://screen/5cjmmZGJKCa. Maybe make these changes in an independent pull request?

**Review by sidharthp — CHANGES_REQUESTED** [review]
Since portfolio/src/main/webapp/favorites.html is empty, move that into a subsequent pull request when you actually add details there?

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (apoorvadhanala) [reply portfolio/src/main/webapp/index.html]: Yeah I chose to make it a start of a new paragraph as I wanted it to start on a new line and be separated from all the content I wrote about.

**Comment by apoorvadhanala** [comment]
I went ahead and changed everything both of you said and created a new pull request(I think) or it may be the same one.

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (yjennywang) [reply portfolio/src/main/webapp/index.html]: This would be the "Click on any of the links..." part, right?

You'll want to wrap that in a new < p > block to get the new break.

**Comment by yjennywang** [comment]
> I went ahead and changed everything both of you said and created a new pull request(I think) or it may be the same one.

This is the same pull request.

To create a new one, you would want to push a different branch and open a new PR.

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (apoorvadhanala) [reply portfolio/src/main/webapp/index.html]: I did do that if you look at my index.html file - not sure why its showing like this here.

**Comment by apoorvadhanala** [comment]
Is there any way you can approve this pull request so I don't have to make a new branch for all the same things? Or can you not?

File: portfolio/src/main/webapp/photography.html (hunk 4 above)
Comment (yjennywang) [reply portfolio/src/main/webapp/photography.html]: Oh, I see. :) this comment is on an older diff!

Looks good, thank you!

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (yjennywang) [reply portfolio/src/main/webapp/index.html]: I see! It's showing as an outdated diff. Looks good, thank you. :)

File: portfolio/src/main/webapp/photography.html (hunk 4 above)
Comment (yjennywang) [reply portfolio/src/main/webapp/photography.html]: this looks resolved by the file deletion :)

**Review by yjennywang — APPROVED** [review]

**Review by sidharthp — APPROVED** [review]
